core/2d/CCNode.cpp

```diff
@@ -525,8 +525,8 @@ void Node::setPosition(float x, float y)
 
 void Node::setPosition3D(const Vec3& position)
 {
-    setPositionZ(position.z);
-    setPosition(position.x, position.y);
+    Node::setPositionZ(position.z);
```

Comment:
Why add prefix `Node::` for these function call?

Reply:
Because Node class calls the override of the camera class when infact it should call it's base function